pyro/infer/reparam/stable.py

```diff
@@ -179,7 +179,7 @@ def __call__(self, name, fn, obs):
         t = _standard_stable(a, one, tu, te, coords="S0")
         a_inv = a.reciprocal()
         skew_abs = fn.skew.abs()
-        t_scale = skew_abs.pow(a_inv)
+        t_scale = skew_abs.clamp(min=torch.finfo(a.dtype).tiny).pow(a_inv)
         s_scale = (1 - skew_abs).pow(a_inv)
```

Comment:
Maybe we should clamp both of these to `.clamp(min=torch.finfo(a.dtype).eps)`?

Reply:
Yes, we can clamp so to prevent the edge case `skew=+-1`. `eps` should be fine too. I was just worried about `grad` at skew=0. Current version returns `grad(skew) = 0.`, which is far from the expected values.